src/render.tsx

```diff
@@ -135,18 +135,18 @@ function updateWithAct(
 }
 
 interface DebugFunction {
-  (message?: string): void;
+  (options?: string | DebugOptions): void;
```

Comment:
Please update flow types (I wish we could drop them^^')

Reply:
Oh I forgot about these, I updated them